README.md

```diff
@@ -12,8 +12,8 @@
 # Webviz subsurface components
 
 `webviz_subsurface_components` is a Dash/React component library for use in `webviz`,
-which have in common that they are geared towards subsurface dashboards. There is an online
-demo available at https://equinor.github.io/webviz-subsurface-components/.
+which have in common that they are geared towards subsurface dashboards. There storybook is available at https://equinor.github.io/webviz-subsurface-components/storybook-static.
```

Comment:
I was thinking to make https://equinor.github.io/webviz-subsurface-components/ as a link to open the storybook and link for demo app to be https://equinor.github.io/webviz-subsurface-components/old-demo to be the link.

I can make the changes if you agree.

Reply:
Sure, sounds good to me.